app/controllers/idv/link_sent_controller.rb

```diff
@@ -46,6 +47,7 @@ def self.step_info
         controller: controller_name,
         next_steps: [:success], # [:ssn],
         preconditions: ->(idv_session:, user:) { idv_session.flow_path == 'hybrid' },
+        undo_step: ->(idv_session:, user:) { true },
```

Comment:
@solipet , @theabrad , what do you think - should this be the same as document_capture's undo_step since it's putting data from the hybrid document_capture into idv_session?